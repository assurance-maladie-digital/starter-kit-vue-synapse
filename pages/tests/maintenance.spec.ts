// @vitest-environment nuxt
import Maintenance from '../maintenance.vue'
import { mountSuspended } from '@nuxt/test-utils/runtime'
import { describe, it, expect } from 'vitest'
import { vuetify } from '@/tests/unit/setup'

describe('maintenance page should render', () => {
	it('should render', async () => {
		const wrapper = await mountSuspended(Maintenance, {
			global: {
				plugins: [vuetify],
			},
		})
		expect(wrapper.html()).toMatchSnapshot()

		const h2Text = wrapper.find('h2').text()
		expect(h2Text).toContain('Maintenance en cours')
	})
})
