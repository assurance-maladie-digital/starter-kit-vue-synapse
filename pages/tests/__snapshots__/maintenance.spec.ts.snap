// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`maintenance page should render > should render 1`] = `"<maintenancepage></maintenancepage>"`;
